src/lib/components/Header.svelte

```diff
@@ -1,15 +1,16 @@
 <script>
-  // your script goes here
+  import logo from "$lib/assets/logos/Wharf-logo-horizontal.svg"
 </script>
 
-<header class="center">
-  <a class="logo" href="/">
-    <img src="/images/wharf-logo.svg" alt="wharf logo" width="56" height="50" />
-    <img src="/images/wharf-wordmark.svg" alt="wharf wordmark" width="129" height="21" />
-  </a>
+<header class="switcher">
+  <div class="logo">
+    <a href="/">
+      <img src={logo} alt="wharf logo" width="203" />
+    </a>
+  </div>
   <nav class="cluster">
-    <!-- <a href="/progress">Progress</a>
-    <a href="/about">Our process</a> -->
+    <a href="/brand">Brandkit</a>
+    <a href="/">Progress</a>
```

Comment:
Do we need this? It does the same thing as clicking the icon currently.

Reply:
Currently not necessary but I thought it looked better having two nav items.

Reply:
Plus, I figure one is explicit and the other is implicit. Once the homepage changes the `Progress` link will remain (if pointing to a different route)